core/src/modules/ops/destination.ts

```diff
@@ -903,15 +902,16 @@ export namespace DestinationOps {
     return { success, error, retryexportIds, retryDelay };
   }
 
-  async function cancelOldExport(oldExport: Export, newExport: Export) {
```

Comment:
We don't have the exact new export now.

Reply:
Noting that this means the cancel messaging is different now too